NEWS.md

```diff
@@ -156,7 +156,7 @@ This major release contains a range of new features and bug fixes that have been
 
 - Documentation updated to reflect changes since version 1.1.0, including new transform and workflow functions.
```

Comment:
No news item for this feature?

Reply:
Good catch. Added a news item in my most recent commit